src/cli/argv_parser.ts

```diff
@@ -227,7 +233,22 @@ const ArgvParser = {
       /* eslint-enable no-console */
     })
 
-    program.parse(argv)
+    try {
+      program.parse(argv)
+    } catch (err) {
+      // An error is always thrown when program.exitOverride() is set.
+      const commanderError: CommanderError = err
```

Comment:
Can we avoid adding this `const` by just adding a type qualifier to the catch arg?

Reply:
Done!